libraries/botbuilder-core/botbuilder/core/bot_assert.py

```diff
@@ -0,0 +1,63 @@
+from typing import List
```

Comment:
Copyright / License statement is missing.

Reply:
Done, actually I just realized that this class was duplicate but this version follows the correct way of none-checking. Removed dup.